### ISSUE TITLE
Markdown support for value description in CLI

### ISSUE THREAD

**Issue by christophenne** [glasskube/glasskube#1022]
**Is your feature request related to a problem? Please describe.**
Markdown is already supported in the description of package configuration values, see #734 

**Describe the solution you'd like**
The `glasskube configure` command should also consider markdown when printing the package config descriptions.

**Comment by shubhrad1** [comment]
Hey @christophenne can I work on this issue

**Comment by Elshirak** [comment]
Hello @christophenne 
I'm interested in working with this task. Could you assign issue to me, please?

**Comment by pmig** [comment]
Hi @Elshirak - How is it going? Did you have already time looking into it?

**Comment by Elshirak** [comment]
Hello @pmig!

Yes, I'm still working on it. Probably, I need to make changes somewhere in <em>./cmd/glasskube/cmd/configure.go</em> file to make __configure__ function support markdown, how it's done in <em>./cmd/glasskube/cmd/describe.go</em>. 

Am I right? Thank you for your attention.

**Comment by pmig** [comment]
@christophenne can you help out here?

**Comment by christophenne** [comment]
@Elshirak yes there is a `printMarkdown` function in `describe.go`. In the `configure` command, there is a user input loop over the configuration values, in which the descriptions are printed – this is where markdown support should be integrated. It makes use of the utility functions in `internal/manifestvalues/cli/configure.go`.

**Comment by aashikantkumar** [comment]
I am fresher for open source projects. sir, anyone can  assign the work

**Comment by pmig** [comment]
@aashikantkumar this ticket is already assigned.

**Comment by aashikantkumar** [comment]
can you assign any work  ?

**Comment by pmig** [comment]
@aashikantkumar I'm sorry but we currently don't have community tasks

**Comment by aashikantkumar** [comment]
How can I contribute to this project?

**Comment by pmig** [comment]
@aashikantkumar please follow https://github.com/glasskube/glasskube/blob/main/CONTRIBUTING.md and keep an eye on good first issues, but currently we don't have any good first issues.

**Comment by aashikantkumar** [comment]
thanks for guide  me

**Comment by bishalbera** [comment]
@christophenne can I take this?

**Comment by christophenne** [comment]
@bishalbera thanks for your interest, but I'd rather reserve this for newcomers to the project, as it's labeled as good first issue.

**Comment by bishalbera** [comment]
@christophenne ohh sorry. I missed that tag

**Comment by NIXBLACK11** [comment]
Hey @christophenne, I would like to work on this issue.

**Comment by NIXBLACK11** [comment]
Hey @pmig, I believe I've resolved the issue. Please review the changes in this [pullrequest](https://github.com/glasskube/glasskube/pull/1174) and let me know if you have any feedback.
